code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts

```diff
@@ -125,21 +125,24 @@ export default async (
       path.join(__dirname, '..', '..', 'templates', 'virtualModuleEntry.template.js')
     );
 
-    previewAnnotations.forEach((previewAnnotationFilename: any) => {
+    previewAnnotations.forEach((previewAnnotationFilename: string | undefined) => {
+      if (!previewAnnotationFilename) return;
```

Comment:
When does this exit early?

Reply:
No idea. Previously it was just being cast to `any` and then used as a string